Temporary Manage Members Mobile Screen for Friday

leon-i opened the pull request:
Members grid hidden on mobile until further refactoring can be made. Still able to add members, displays message to use desktop if you want to see members grid. Pic below:  
  
![image](https://user-images.githubusercontent.com/56734437/90151327-99782280-dd54-11ea-93c3-355ddd37ee4f.png)  
  
Also added link to manage members screen back to mobile nav menu.

ibeckermayer commented on frontend/client/styles/screens/manage_teams.scss:
```diff
@@ -12,11 +17,25 @@
 }
 
+.mobile-msg {
+  display: none;
```
Nitpick here:

Let's make this
```scss
@include media('>phone') {
    display: none;
}
```

As a general rule I think its much easier to read this conditional scss code when it's explicit about what's for mobile and what's for desktop. I've found when I don't write the code this way, I commonly start mixing the paradigm up (i.e. some classes the default is mobile, while others the default is desktop), and code integrity degrades over time.

leon-i replied:
Fixed, no prob - makes sense

leon-i commented:
Fixed role select overflow, adjusted the height of the modal on mobile to better fit because it was overflowing a bit on smaller mobile devices (iPhone 5/5s). Pics below:  
  
Iphone X:  
![image](https://user-images.githubusercontent.com/56734437/90169021-00550600-dd6c-11ea-8f0e-55a76fb329c9.png)  
  
Iphone 5/5s:  
![image](https://user-images.githubusercontent.com/56734437/90169038-08ad4100-dd6c-11ea-971a-da0686bf5834.png)

leon-i commented:
Fixed it so the Role select dropdown arrow now appears on smaller mobile devices

leon-i commented:
Merged #488 into `dev`.
